MaxText/configs/models/gpt3-52k.yml

```diff
@@ -37,3 +37,4 @@ adam_b1: 0.9
 adam_b2: 0.95
 adam_eps: 1.e-8
 adam_weight_decay: 0.1
+attention: "dot_product"  # head_dim 8 is too small for splash/flash attention
```

Comment:
I recall Rafi suggested us to put doc_product to the end_to_end test script, instead of model yml.

Reply:
We need it in checkpoint conversion as well when we compile and create initial weights. Therefore, changing the default model setting would be the safest.

Reply:
+1 to @RissyRan and +2 to @ZhiyuLi-goog